client-misk-hibernate/src/test/kotlin/app/cash/backfila/client/misk/hibernate/SinglePartitionHibernateTestBackfill.kt

```diff
@@ -20,7 +20,7 @@ class SinglePartitionHibernateTestBackfill @Inject constructor(
   val parametersLog = mutableListOf<SandwichParameters>()
 
   override fun backfillCriteria(config: BackfillConfig<SandwichParameters>): Query<DbMenu> {
-    return queryFactory.newQuery(MenuQuery::class).name(config.parameters.type)
+    return queryFactory.newQuery(MenuQuery::class).name(config.parameters.type ?: "chicken")
```

Comment:
Maybe instead of chicken for both, null should maybe be something else like beef?

Reply:
done, that makes it clearer that it uses `chicken` when null is passed, which is kinda weird, but better than before this PR where we would pass "null"